### ISSUE TITLE
prefer-bimap false positive warning

### ISSUE THREAD

**Issue by giogonzo** [buildo/eslint-plugin-fp-ts#47]
Just saw this in the eslint logs:

```
warning: mapLeft followed by mapLeft can be replaced by bimap
```

Which is not right 😅 Could it be a false positive (it should warn for map+mapLeft, or for mapLeft+map, but not for mapLeft+mapLeft)

**Comment by gabro** [comment]
Thanks! Fixed in v0.1.14 🚀
